<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>113</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>113</y>
     <width>1920</width>
     <height>967</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8e8e8; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>14</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>147</y>
     <width>1800</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
    border: 2px solid #ffffff;
    border-radius: 30px; /* Bordes redondeados */
    padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>175</y>
     <width>278</width>
     <height>63</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);
color: white;

border: 2px solid #002d6b;
border-radius: 15px; /* Bordes redondeados */
padding: 5px;



</string>
   </property>
   <property name="text">
    <string> Agregar Empleados</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../c.png</normaloff>../c.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>36</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>RECURSOS HUMANOS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>36</y>
     <width>126</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1836</x>
     <y>29</y>
     <width>56</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../perfil-de-usuario.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>190</y>
     <width>355</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Empleados: {total}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>255</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>385</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>515</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>645</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>710</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>775</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>840</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>970</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 45, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>905</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>262</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);

border: 2px solid #00000;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>#0001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>328</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);

border: 2px solid #00000;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>#0001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>393</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);

border: 2px solid #00000;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>#0001</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>262</y>
     <width>248</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8e8e8;
color: black;

border: 2px solid #e8e8e8;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>Chingui Lin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>328</y>
     <width>248</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8e8e8;
color: black;

border: 2px solid #e8e8e8;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>Bebe Lin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>393</y>
     <width>248</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8e8e8;
color: black;

border: 2px solid #e8e8e8;
border-radius: 25px; /* Bordes redondeados */
padding: 5px;
</string>
   </property>
   <property name="text">
    <string>Choco Lin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1734</x>
     <y>266</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;

border: 2px solid black;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../edit.png</normaloff>../edit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1734</x>
     <y>331</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;

border: 2px solid black;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../edit.png</normaloff>../edit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1734</x>
     <y>399</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;

border: 2px solid black;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../edit.png</normaloff>../edit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1791</x>
     <y>331</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;

border: 2px solid black;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../close.png</normaloff>../close.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1791</x>
     <y>266</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;

border: 2px solid black;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../close.png</normaloff>../close.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>1791</x>
     <y>399</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>icon-ui</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;

border: 2px solid black;
border-radius: 20px; /* Bordes redondeados */
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../close.png</normaloff>../close.png</iconset>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_7</zorder>
  <zorder>label_14</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_9</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>label_18</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>label_21</zorder>
  <zorder>label_35</zorder>
  <zorder>label_36</zorder>
  <zorder>label_37</zorder>
  <zorder>label_38</zorder>
  <zorder>label_39</zorder>
  <zorder>label_40</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
